NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -151,6 +152,18 @@ class NeuroID
                 this.getApplicationContext()
                     ?.let { nidCallActivityListener?.setCallActivityListener(it) }
             }
+
+            //get connectivity info on startup
+            application?.let {
+                val connectivityManager =
+                    it.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+                val info = connectivityManager.activeNetworkInfo
+                val isConnectingOrConnected = info?.isConnectedOrConnecting()
+                if (isConnectingOrConnected != null) {
+                    isConnected = isConnectingOrConnected
+                }
+                isWifi = info?.type == ConnectivityManager.TYPE_WIFI
```

Comment:
is wifi enough here? do we want to get the specifics or wifi vs. cell vs. ethernet?

Reply:
yeah I think that would be good. ill change this to string and add the type directly there.